typescript/src/accounts/evm/networkCapabilities.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Centralized configuration for network capabilities.
+ * This defines which methods are available on which networks.
+ */
+
+/**
+ * Network capabilities configuration.
+ * Each network has a set of boolean flags indicating which methods are supported.
+ */
+export const NETWORK_CAPABILITIES = {
+  base: {
+    listTokenBalances: true,
+    requestFaucet: false,
+    quoteFund: true,
+    fund: true,
+    waitForFundOperationReceipt: true,
+    transfer: true,
+    sendTransaction: true,
+    quoteSwap: true,
+    swap: true,
+  },
+  "base-sepolia": {
+    listTokenBalances: true,
+    requestFaucet: true,
+    quoteFund: false,
+    fund: false,
+    waitForFundOperationReceipt: false,
+    transfer: true,
+    sendTransaction: true,
+    quoteSwap: false,
+    swap: false,
+  },
+  ethereum: {
+    listTokenBalances: true,
+    requestFaucet: false,
+    quoteFund: false, // Only base is supported for quoteFund
+    fund: false, // Only base is supported for fund
+    waitForFundOperationReceipt: false,
+    transfer: false, // Not supported based on SendEvmTransactionBodyNetwork
+    sendTransaction: true, // Always available (uses wallet client for non-base networks)
```

Comment:
This code comment will become untrue once [this PR](https://github.com/coinbase/cdp-sdk/pull/234) lands, at which point we can update the network hoisting feature to use CDP API for tx sends on eth mainnet and sepolia